src/Statiq.Web.Templates/Statiq.Web.Templates.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <PackageId>Statiq.Web.Templates</PackageId>
+    <Title>Statiq Web Templates</Title>
+    <Authors>Dave Glick</Authors>
+    <Description>Templates to use when creating a Statiq Web application, Statiq Web is a flexible static site generator.</Description>
+    <PackageTags>dotnet-new;templates;Statiq;Static;StaticContent;StaticSite;Blog;BlogEngine </PackageTags>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IncludeContentInPack>true</IncludeContentInPack>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IncludeSymbols>false</IncludeSymbols>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <ContentTargetFolders>content</ContentTargetFolders>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="templates\**\*" Exclude="templates\**\bin\**;templates\**\obj\**;templates\**\.vs\**" />
+    <Compile Remove="**\*" />
+  </ItemGroup>
+
+  <Target Name="VersionBuild" BeforeTargets="PrepareForBuild">
+   <XmlPoke XmlInputPath="templates\statiq-web\StatiqWeb\StatiqWeb.csproj" Query="Project/ItemGroup/PackageReference/@Version" Value="$(Version)" />
```

Comment:
This will ensure packed template references same package as built solution.

Reply:
🎉 great! That way a new version of the template will just roll out with each release, right?

(or at least the build of the template will reference the currently releasing version, I'll probably need to add the last mile deployment of the template to the build script)

Reply:
Correct it'll always be in sync with the version you're building.